### PR TITLE
added ansible-splitter for nxos

### PR DESCRIPTION
**Pull request by komaldesai13**

### CODE REVIEW

**Comment by komaldesai13** [comment]
recheck

**Comment by komaldesai13** [comment]
recheck

**Comment by komaldesai13** [comment]
recheck

File: zuul.d/cisco-nxos-jobs.yaml [hunk 1]
```diff
@@ -66,3 +71,55 @@
     parent: ansible-ee-integration-cisco-nxos
     vars:
       test_ansible_skip_tags: local,nxapi
+
+- job:
+    name:integration-ciso.nxos-cli-python38-latest-1
+    parent:ansible-ee-integration-cisco-nxos-cli-python38-latest
```
Comment (NilashishC) [inline zuul.d/cisco-nxos-jobs.yaml:77]: Shouldn't there be a space between `parent:` and the job name?

File: zuul.d/project-templates.yaml [hunk 2]
```diff
@@ -374,12 +374,28 @@
...
+        - ansible-test-splitter:
             required-projects:
               - name: github.com/ansible-collections/ansible.netcommon
               - name: github.com/ansible-collections/ansible.utils
+            vars:
+              ansible_test_collections: true
+              ansible_test_splitter__test_changed: false
+              ansible_test_splitter__check_for_changes_in:
+                - "~/{{ zuul.projects['github.com/ansible-collections/cisco.nxos'].src_dir }}"
+        - integration-ciso.nxos-cli-python38-latest-1
```
Comment (NilashishC) [inline zuul.d/project-templates.yaml:386]:
```suggestion
        - integration-cisco.nxos-cli-python38-latest-1
```

File: zuul.d/cisco-nxos-jobs.yaml [hunk 3]
```diff
@@ -66,3 +71,55 @@
...
+- job:
+    name: integration-ciso.nxos-cli-python38-latest-11
+    parent: ansible-ee-integration-cisco-nxos-cli-python38-latest
+
+- job:
+    name: integration-ciso.nxos-cli-python38-latest-12
+    parent: ansible-ee-integration-cisco-nxos-cli-python38-latest
+
+- job:
+    name: integration-ciso.nxos-cli-python38-latest--13
```
Comment (NilashishC) [inline zuul.d/cisco-nxos-jobs.yaml]:
```suggestion
    name: integration-ciso.nxos-cli-python38-latest-13
```

**Pull request closed without merging**